Test failing on x86_64-darwin :(

```
test cases/osx/9 framework recasting/meson.build:3:4: ERROR: Dependency "openal" not found, tried pkgconfig and framework
 
Traceback (most recent call last):
  File "/private/tmp/nix-build-meson-1.4.0.drv-0/source/run_tests.py", line 318, in run_configure_inprocess
    returncode = mesonmain.run(commandlist, get_meson_script())
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/private/tmp/nix-build-meson-1.4.0.drv-0/source/mesonbuild/mesonmain.py", line 277, in run
    return CommandLineParser().run(args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/private/tmp/nix-build-meson-1.4.0.drv-0/source/mesonbuild/mesonmain.py", line 188, in run
    return errorhandler(e, command)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/private/tmp/nix-build-meson-1.4.0.drv-0/source/mesonbuild/mesonmain.py", line 31, in errorhandler
    raise e
  File "/private/tmp/nix-build-meson-1.4.0.drv-0/source/mesonbuild/mesonmain.py", line 186, in run
    return options.run_func(options)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/private/tmp/nix-build-meson-1.4.0.drv-0/source/mesonbuild/msetup.py", line 363, in run
    app.generate()
  File "/private/tmp/nix-build-meson-1.4.0.drv-0/source/mesonbuild/msetup.py", line 187, in generate
    return self._generate(env, capture, vslite_ctx)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/private/tmp/nix-build-meson-1.4.0.drv-0/source/mesonbuild/msetup.py", line 226, in _generate
    intr.run()
  File "/private/tmp/nix-build-meson-1.4.0.drv-0/source/mesonbuild/interpreter/interpreter.py", line 3007, in run
    super().run()
  File "/private/tmp/nix-build-meson-1.4.0.drv-0/source/mesonbuild/interpreterbase/interpreterbase.py", line 162, in run
    self.evaluate_codeblock(self.ast, start=1)
  File "/private/tmp/nix-build-meson-1.4.0.drv-0/source/mesonbuild/interpreterbase/interpreterbase.py", line 188, in evaluate_codeblock
    raise e
  File "/private/tmp/nix-build-meson-1.4.0.drv-0/source/mesonbuild/interpreterbase/interpreterbase.py", line 180, in evaluate_codeblock
    self.evaluate_statement(cur)
  File "/private/tmp/nix-build-meson-1.4.0.drv-0/source/mesonbuild/interpreterbase/interpreterbase.py", line 198, in evaluate_statement
    self.assignment(cur)
  File "/private/tmp/nix-build-meson-1.4.0.drv-0/source/mesonbuild/interpreterbase/interpreterbase.py", line 634, in assignment
    value = self.evaluate_statement(node.value)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/private/tmp/nix-build-meson-1.4.0.drv-0/source/mesonbuild/interpreterbase/interpreterbase.py", line 194, in evaluate_statement
    return self.function_call(cur)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/private/tmp/nix-build-meson-1.4.0.drv-0/source/mesonbuild/interpreterbase/interpreterbase.py", line 520, in function_call
    res = func(node, func_args, kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/private/tmp/nix-build-meson-1.4.0.drv-0/source/mesonbuild/interpreterbase/decorators.py", line 799, in wrapped
    return f(*wrapped_args, **wrapped_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/private/tmp/nix-build-meson-1.4.0.drv-0/source/mesonbuild/interpreterbase/decorators.py", line 799, in wrapped
    return f(*wrapped_args, **wrapped_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/private/tmp/nix-build-meson-1.4.0.drv-0/source/mesonbuild/interpreterbase/decorators.py", line 799, in wrapped
    return f(*wrapped_args, **wrapped_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  [Previous line repeated 4 more times]
  File "/private/tmp/nix-build-meson-1.4.0.drv-0/source/mesonbuild/interpreterbase/decorators.py", line 98, in wrapped
    ret = f(*wrapped_args, **wrapped_kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/private/tmp/nix-build-meson-1.4.0.drv-0/source/mesonbuild/interpreterbase/decorators.py", line 116, in wrapped
    return f(*wrapped_args, **wrapped_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/private/tmp/nix-build-meson-1.4.0.drv-0/source/mesonbuild/interpreterbase/decorators.py", line 250, in wrapper
    return f(*nargs, **wrapped_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/private/tmp/nix-build-meson-1.4.0.drv-0/source/mesonbuild/interpreterbase/decorators.py", line 569, in wrapper
    return f(*wrapped_args, **wrapped_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/private/tmp/nix-build-meson-1.4.0.drv-0/source/mesonbuild/interpreter/interpreter.py", line 1787, in func_dependency
    d = df.lookup(kwargs)
        ^^^^^^^^^^^^^^^^^
  File "/private/tmp/nix-build-meson-1.4.0.drv-0/source/mesonbuild/interpreter/dependencyfallbacks.py", line 355, in lookup
    dep = func(kwargs, func_args, func_kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/private/tmp/nix-build-meson-1.4.0.drv-0/source/mesonbuild/interpreter/dependencyfallbacks.py", line 87, in _do_dependency
    dep = dependencies.find_external_dependency(name, self.environment, kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/private/tmp/nix-build-meson-1.4.0.drv-0/source/mesonbuild/dependencies/detect.py", line 165, in find_external_dependency
    raise DependencyException(f'Dependency "{name}" not found' +
mesonbuild.dependencies.base.DependencyException: Dependency "openal" not found, tried pkgconfig and framework
```

--- a/pkgs/by-name/me/meson/package.nix
+++ b/pkgs/by-name/me/meson/package.nix
@@ -14,17 +14,17 @@
 }:
 
 let
-  inherit (darwin.apple_sdk.frameworks) AppKit Cocoa Foundation LDAP OpenGL;
+  inherit (darwin.apple_sdk.frameworks) AppKit Cocoa Foundation LDAP OpenAL OpenGL;
 in
 python3.pkgs.buildPythonApplication rec {
   pname = "meson";
-  version = "1.3.2";
+  version = "1.4.0";
 
   src = fetchFromGitHub {
     owner = "mesonbuild";
     repo = "meson";
     rev = "refs/tags/${version}";
-    hash = "sha256-7M/El2snWsQi+gaZWPHnEr9gpJW3trqG1RbnT43M49s=";
+    hash = "sha256-hRTmKO2E6SIdvAhO7OJtV8dcsGm39c51H+2ZGEkdcFY=";
   };
 
   patches = [
@@ -86,6 +86,7 @@ python3.pkgs.buildPythonApplication rec {
     Cocoa
     Foundation
     LDAP
+    OpenAL
     OpenGL
     openldap
   ];
@@ -96,6 +97,7 @@ python3.pkgs.buildPythonApplication rec {
       patchShebangs 'test cases'
       substituteInPlace \
         'test cases/native/8 external program shebang parsing/script.int.in' \
+        'test cases/common/273 customtarget exe for test/generate.py' \
           --replace /usr/bin/env ${coreutils}/bin/env
     ''
   ]

